README.md

```diff
@@ -28,6 +31,17 @@ cd beavsAI-backend
 npm install
 ```
 
+4. Get the environment variables
+
+Create a `.env` file in the `/config/` directory under `.env.local` and add the following variables:
+```sh
```

Comment:
```suggestion
```dotenv
```

Reply:
Wait that's so cool that linguist supports `dotenv` highlighting - good catch